src/components/Editor/CodeEditor.css

```diff
@@ -18,3 +18,26 @@
   user-select: none;
   cursor: default;
 }
+
+.codePair-cursor {
+  position: absolute;
+  border-left-style: solid;
+  border-left-width: 2.5px;
+  padding: 0px;
+}
+
+.codePair-name {
+  position: absolute;
+  left: -2px;
+  padding: 1px 4px;
+  border-radius: 2px 2px 2px 0px;
+}
```

Comment:
How about creating a CodePair.css file and using a simple name in it?

```
.cursor {
  position: absolute;
  border-left-style: solid;
  border-left-width: 2.5px;
  padding: 0px;
}

.name {
  position: absolute;
  left: -2px;
  padding: 1px 4px;
  border-radius: 2px 2px 2px 0px;
}
```

Reply:
It is already existed named CodeEditor.css

you mean just want to change `codePair-name` to `name` ?

I think we don't use `sass` yet so If we use the name, It is not dependent on one file. It would apply for all file

Reply:
The advantage of react is componentization.
Different styles can be applied for each component.

e.g. 
CodeEditor.css for CodeEditor.tsx,
ClientCursor.css for ClientCursor.tsx

and I think the simpler the name, the better. 😄

Reply:
yes you are right!

but we don't prepare to work about `The advantage of react is componentization.` yet
So, if you use the style name as `name`, it is applied globally.

We haven't been working on the style module yet, after this we can use it as you said!